src/routes/transactions/mappers/common/twap-order.mapper.spec.ts

```diff
@@ -841,6 +948,7 @@ describe('TwapOrderMapper', () => {
     });
 
     expect(result).toEqual({
+      activeOrderUid: null,
       buyAmount: '51576509680023161648',
       buyToken: {
         address: buyToken.address,
```

Comment:
Should we add these (or similar) testcases for the Order Status mapping?

- Cancelled Order with 3 parts where only the first part exists and the second part is currently active.
- Fulfilled Order with 3 parts where the last part is currently active and the repository returns fulfilled for it.

Reply:
Added in f20a158137e033f1620ec10484a6501a82057e7f.